src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWebCSharp.1/Components/Account/Pages/ResetPassword.razor

```diff
@@ -90,11 +96,13 @@
     {
         [Required]
         [EmailAddress]
+        [Display(Name = "Email")]
```

Comment:
How can we know this is working if the property name is the same as the given Name?

Reply:
This is template. We used to hardcode these values, the PR just changes it to dynamically revolved values. Originally, email used to be just email:
`<label for="Input.Email" class="form-label">Email</label>`
If you want to see a more suitable case, check out e.g. "ConfirmPassword". It is `Display(Name = "Confirm new password")]`.